mods/ra/rules/defaults.yaml

```diff
@@ -249,8 +249,14 @@
 	Selectable:
 		Bounds: 24, 24
 	Targetable:
-		TargetTypes: Ground, Repair, Vehicle
-		RequiresCondition: !parachute
+		TargetTypes: Ground, Vehicle, Repair
+		RequiresCondition: !parachute && !undamaged
+	Targetable@UNDAMAGED:
+		TargetTypes: Ground, Vehicle
+		RequiresCondition: !parachute && undamaged
```

Comment:
We support multiple active `Targetable` traits, right?
IMO it would be cleaner to keep this targetable active all the time, and then make the other one only add `Repair` when needed.